### ISSUE TITLE
Unable to access bin_algo

### ISSUE THREAD

**Issue by virsto** [secnot/rectpack#24]
I have download and installed this package but found that I could not change bin_algo and sort_algo. I did get sort_algo to work (able to change it) by moving the lamba functions to the main program; but have been uable to access/change bin_algo.

**Comment by secnot** [comment]
Most probably is an import problem, could you provide a simple code snippet where it fails?

Thank you

**Comment by virsto** [comment]
Yes, Secnot
Here is some code that used for a simple box (without a top). I plan to construct such a box from an arcrylic sheet and thought it would be interesting to see how rectpack would arrange them on the sheet. And thanks very much for putting your package into GitHub.
[RectanglePack_02.txt](https://github.com/secnot/rectpack/files/2522029/RectanglePack_02.txt)

**Comment by virsto** [comment]
I should not have closed this!!

**Comment by virsto** [comment]
Yes! My import was rather limited --- I should have seen this. Keep up the good work. It would be interesting to know more about how to use rectpack to make guillotine cuts.

**Comment by secnot** [comment]
The guillotine algorithm implementation doesn't track the start and end of each cut.

You can deduce them using the packing result, or patch the algorithm to store the cut each time a rectangle is placed.
